contracts/Elections.sol

```diff
@@ -158,113 +157,85 @@ contract Elections is IElections, ContractRegistryAccessor, WithClaimableFunctio
 		}
 	}
 
-	function voteOut(address[] calldata subjectAddrs) external onlyWhenActive {
-		require(subjectAddrs.length <= 3, "up to 3 concurrent votes are supported");
-		for (uint i = 0; i < subjectAddrs.length; i++) {
-			require(subjectAddrs[i] != address(0), "all votes must non zero addresses");
+	function voteOut(address subjectAddr) external onlyWhenActive {
+		Settings memory _settings = settings;
+
+		address prevSubject = voteOutVotes[msg.sender];
+		voteOutVotes[msg.sender] = subjectAddr;
+
+		if (prevSubject == address(0)) {
+			votersStake[msg.sender] = getDelegationsContract().getDelegatedStakes(msg.sender);
+		}
+
+		uint256 voterStake = votersStake[msg.sender];
+		uint totalStake = getDelegationsContract().getTotalDelegatedStake();
+
+		if (prevSubject != address(0) && prevSubject != subjectAddr) {
+			accumulatedStakesForVoteOut[prevSubject] = accumulatedStakesForVoteOut[prevSubject].sub(voterStake);
+			_applyVoteOutVotesFor(prevSubject, totalStake, _settings);
+		}
+
+		if (subjectAddr != address(0)) {
```

Comment:
Cleaner to clear votersStake[msg.sender] =. 0 when clearing the vote (subjectAddr == address(0))